File: docs/apis/reference/snippets/java/tts/voice_generate.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("application/json");
RequestBody body = RequestBody.create(mediaType, "{\n    \"text\": \"Hello, Surreal Engine\",\n    \"voice_name\": \"en-US-Wavenet-A\"\n}");
Request request = new Request.Builder()
  .url("https://openapi.surreal-ai.com/v1/tts/voice.generate")
  .method("POST", body)
  .addHeader("Authorization", "bee03bed6352400383d9a51ba325af2f")
  .addHeader("Content-Type", "application/json")
  .build();
Response response = client.newCall(request).execute();

File: docs/reference/snippets/javascript/tts/voice_generate.js
var myHeaders = new Headers();
myHeaders.append("Authorization", "bee03bed6352400383d9a51ba325af2f");
myHeaders.append("Content-Type", "application/json");

var raw = JSON.stringify({
  "text": "Hello, Surreal Engine",
  "voice_name": "en-US-Wavenet-A"
});

var requestOptions = {
  method: 'POST',
  headers: myHeaders,
  body: raw,
  redirect: 'follow'
};

fetch("https://openapi.surreal-ai.com/v1/tts/voice.generate", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));
